.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
What I meant - should this file be part of the repository or should it be owned by a developer (hence `.gitignore`d)